<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scientistadd</class>
 <widget class="QDialog" name="scientistadd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="scientistLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add new scientist</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dyLine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dyLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year of death:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="maleRadio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>102</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QLabel" name="byLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year of birth:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="genderLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="byLine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="femaleRadio">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>102</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
